antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/SqlBase.g4

```diff
@@ -1370,6 +1281,10 @@ MINUS : '-';
 
 PLUS : '+';
 
+DIV : '/';
+
+MOD : M O D | '%';
```

Comment:
`MOD` keywords are not recommended, which makes it necessary to reserve more keywords

Reply:
I removed the `M O D`.